### DISCUSSION TITLE
Ability to share URLs to specific files/lines in PRs

### DISCUSSION THREAD

**Discussion by rafasoares** [community/community#73045] [Mobile]
When sharing links to PRs externally, sometimes we want to point to a specific file/line/diff. We already have that ability on the web version. 

Tap and hold on a file header or a line gutter seems like a very intuitive approach. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.136.1 (113857946)
Device: iPhone 12 (iOS 17.0.3)
</details>

**Comment by oussamaef06** [comment]
GitHub for iOS did not support direct linking to specific files, lines, or diffs within pull requests. GitHub's mobile apps have been more limited in terms of deep linking and file-level interactions compared to the web interface.

However, GitHub continually updates its mobile apps to enhance their functionality. It's possible that such a feature might have been added in a more recent version or will be added in future updates.

To know if this feature has been added since my last update, I recommend checking the official release notes or the GitHub for iOS app's documentation on the App Store for any new features or updates related to linking to specific files, lines, or diffs within pull requests. You can also explore the app's settings and look for any new functionality related to sharing or linking to specific content within pull requests.
